File: php/difference-of-squares/difference-of-squares.php
<?php

function squareOfSums($num) {
    // https://www.google.de/search?q=sum+of+sequence
    $sum = $num * (1 + $num) / 2;

    return $sum * $sum;
}

function sumOfSquares($num) {
    // http://www.trans4mind.com/personal_development/mathematics/series/sumNaturalSquares.htm
    return $num * ($num + 1) * (2 * $num + 1) / 6;
}

function difference($num) {
    return squareOfSums($num) - sumOfSquares($num);
}

File: go/difference-of-squares/difference_of_squares.go
package diffsquares

func SquareOfSums(num int) int {
	// https://www.google.de/search?q=sum+of+sequence
	sqsum := num * (1 + num) / 2

	return sqsum * sqsum
}

func SumOfSquares(num int) int {
	// http://www.trans4mind.com/personal_development/mathematics/series/sumNaturalSquares.htm
	return num * (num + 1) * (2*num + 1) / 6
}

func Difference(num int) int {
	return SquareOfSums(num) - SumOfSquares(num)
}
